enacts/calc.py

```diff
@@ -619,6 +620,112 @@ def daily_tobegroupedby_season(
     return daily_tobegroupedby_season
 
 
+def assign_season_coords(
+    daily_data,
```

Comment:
called it time_series. Indeed as I look a the code, it doesn't look like it would care that the data is dekadal or monthly. It's not like xarray understands so much about things being daily anyways.
We might want to get to the bottom of this then and make start_/end_day optional because it would be awkward to ask for them in the "monthly" case